client-side.md

```diff
@@ -96,12 +74,6 @@ various areas below.
 
 - Don’t fail silently.
 - Report your JavaScript errors into a logging system (e.g. Sentry).
-
-*TODO: Would be good to have something about performance (memory,
-responsiveness, etc), maybe a strategy to assess performance, some
-targets, automated testing or RUM?*
+- Define web performance budgets and track them
 
 ### Build tools/task runners
-
-- Have an automated process to produce your final assets (e.g. using
-  Grunt, Gulp, [Plumber](https://github.com/plumberjs/plumber), etc).
```

Comment:
Why remove this one? The example tooling is a bit dated, admittedly.

Reply:
👍  Have updated with a more modern tool list.